joblib/memory.py

```diff
@@ -10,33 +10,29 @@
 
 
 from __future__ import with_statement
+
+import functools
+import inspect
+import io
+import json
```

Comment:
It doesn't seem like there is a good reason for this file to be modified. Am I missing something?